### DISCUSSION TITLE
How are the packages installed?

### DISCUSSION THREAD

**Discussion by katastrophe92** [chocolatey/choco#3295] [Q&A]
Hello Chocolatey community! Maybe my question is dumb 😆 ... installed packages are portable like for example, packages installed through Portable apps or like .exe apps? Hope someone understands what I mean 😂 
Hope someone can help. Thanks!

**Comment by pauby** [comment]
Have you had a look at the [docs](https://docs.chocolatey.org)?

**Comment by katastrophe92** [comment]
I couldn't find it where it says how are softwares installed. Only that ''Packages are Independent and Portable.'' but do not know if it the proper answer

**Reply by TheCakeIsNaOH** [reply]
I'd suggest reading through a couple FAQs here: https://docs.chocolatey.org/en-us/faqs#what-distinction-does-chocolatey-make-between-an-installable-and-a-portable-application
